docs/pages/deploy-a-cluster/helm-deployments/azure.mdx

```diff
@@ -0,0 +1,335 @@
+$ az role assignment create --role "Storage Blob Data Owner" --assignee-principal-type ServicePrincipal \
+    --assignee-object-id "$(az identity show --resource-group <Var name="teleport-rg" /> --name teleport-id --query principalId -o tsv)" \
+    --scope "$(az storage account show --resource-group <Var name="teleport-rg" /> --name <Var name="teleportblob" /> --query id -o tsv)""
+```
+
+We'll use Workload Identity to authenticate to those services, so we'll add
+federated credentials for the Teleport service account used by the Auth Service.
+
+```code
+$ az identity federated-credential create --resource-group <Var name="teleport-rg" /> --identity-name teleport-id \
+    --name aks --audience api://AzureADTokenExchange \
+    --subject system:serviceaccount:<Var name="teleport-ns" />:<Var name="teleport-release" /> \
+    --issuer "$(az aks show --resource-group <Var name="aks-rg" /> --name <Var name="aks-name" /> --query oidcIssuerProfile.issuerUrl -o tsv)"
+```
+
+## Step 3/5. Set values to configure the cluster
+
+<Details title="License Secret" scopeOnly={false} scope={"enterprise"} opened={true}>
+
+Before you can install Teleport Enterprise in your Kubernetes cluster, you will need to
+create a secret that contains your Teleport license information.
+
+(!docs/pages/includes/enterprise/obtainlicense.mdx!)
+
+Create a secret from your license file. Teleport will automatically discover
+this secret as long as your file is named `license.pem`.
+
+```code
+$ kubectl create namespace <Var name="teleport-ns" />
+$ kubectl -n <Var name="teleport-ns" /> create secret generic license --from-file=license.pem
+```
+
+</Details>
+
+Now we'll configure the `teleport-cluster` Helm chart to use the `azure` mode.
+
+First get the client ID for the `teleport-id` identity:
+```code
+$ az identity show --resource-group <Var name="teleport-rg" /> --name teleport-id --query clientId -o tsv
+<Var name="teleport-clie-ntid-uuid-123456789012" />
+```
+
+Then create a file called `azure-values.yaml` containing the values you've selected above:
+
+<Tabs>
+<TabItem scope={["oss"]} label="Open Source">
+
+```yaml
+chartMode: azure
+# Name of your cluster. Use the FQDN you intend to configure in DNS later
+clusterName: teleport.example.com
+azure:
+  databaseHost: "<Var name="teleport-pg" />.postgres.database.azure.com"
+  databaseUser: "<Var name="pguser" />"
+  sessionRecordingStorageAccount: "<Var name="teleportblob" />.blob.core.windows.net"
+  # Whether to mirror audit log entries to stdout in JSON format (useful for external log collectors)
+  auditLogMirrorOnStdout: false
+  clientID: "<Var name="teleport-clie-ntid-uuid-123456789012" />"
+highAvailability:
+  # Number of replicas to configure
+  replicaCount: 2
+  certManager:
+    # Enable cert-manager support to get TLS certificates
+    enabled: true
+    # Name of the cert-manager GlobalIssuer or Issuer to use
+    issuerName: letsencrypt-production
+    issuerKind: GlobalIssuer
```

Comment:
I think this should be `ClusterIssuer` as per https://cert-manager.io/docs/concepts/issuer/?

```suggestion
    issuerKind: ClusterIssuer
```

Reply:
It should!